python/example/ring.py

```diff
@@ -48,18 +48,32 @@ def event_generators(self, gid):
 context = arbor.context(threads=4, gpu_id=None)
 print(context)
 
+meters = arbor.meter_manager()
+print(meters)
```

Comment:
The meters don't need to be printed here because they are currently empty.

Reply:
Done.

Reply:
Also changed prints to python3 format printing standard.